api/build.gradle

```diff
@@ -404,15 +405,16 @@ dependencies {
     compile "io.opencensus:opencensus-impl:$project.ext.OPENCENSUS_VERSION"
     compile 'io.swagger:swagger-annotations:1.6.6'
     compile "javax.inject:javax.inject:1"
-    compile 'org.liquibase:liquibase-core:4.9.1'
+    compile "org.liquibase:liquibase-core:$project.ext.LIQUIBASE_VERSION"
     compile 'mysql:mysql-connector-java:8.0.29'
     compile "org.apache.commons:commons-collections4:4.4"
     compile 'org.apache.commons:commons-lang3:3.12.0'
     compile 'org.apache.commons:commons-text:1.9'
     compile "org.apache.httpcomponents:httpclient:4.5.13"
     compile 'org.springframework.boot:spring-boot-starter-validation'
     compile "commons-codec:commons-codec:1.15"
-    compile 'com.auth0:java-jwt:3.19.1'
+    compile 'com.auth0:java-jwt:3.19.2'
+    compile 'io.netty:netty-common:4.1.77.Final'
```

Comment:
The new `google-cloud-bigquery` brings in a vulnerable version of this package, so we now need to upgrade this as well.

https://app.snyk.io/org/all-of-us-researcher-workbench/project/bf2628cc-574b-4c92-a236-237ad192de46/pr-check/2a223ccd-ba08-4222-88ed-585dd5444d4c

Reply:
👍🏻